setup.py

```diff
@@ -14,7 +14,8 @@
     'six',
     'pytz',
     'pandas',
-    'jsonpath_ng'
+    'jsonpath_ng',
+    'bpanonymize @ git+https://kcho@github.com/AMP-SCZ/dpanonymize@v0.1#egg=bpanonymize'
```

Comment:
I inserted missing `v` [here](https://github.com/AMP-SCZ/dpanonymize/releases/tag/v0.1)

Reply:
I only know about this syntax:

> pip install git+https://github.com/aladagemre/django-notification.git@v0.1

Have you tested Ln 18 as written?